wds-required-plugins.php

```diff
@@ -112,11 +112,6 @@ public function required_text_markup() {
 	 * @return array
 	 */
 	public function filter_plugin_links( $actions = array(), $plugin ) {
-		// Remove edit link for all plugins
-		if ( array_key_exists( 'edit', $actions ) ) {
-			unset( $actions['edit'] );
-		}
-
```

Comment:
I'm cool w/ either way, and will bow to a 2-1 vote, but IMO, if I don't want a client to deactivate a plugin, (because it would break their site), then I sure as heck don't want them editing it.